railties/test/commands/server_test.rb

```diff
@@ -108,4 +108,14 @@ def test_log_stdout
       end
     end
   end
+
+  def test_default_options
+    server = Rails::Server.new
+    old_default_options = server.default_options
+
+    # Regression test for issue #22811
```

Comment:
Can you remove this comment? Your commit message is great, we don't need the extra comment because a git blame will find the commit and the bug it fixed. Thanks!